qt/applications/workbench/workbench/plotting/toolbar.py

```diff
@@ -30,27 +30,27 @@ class WorkbenchNavigationToolbar(NavigationToolbar2QT):
     sig_waterfall_conversion = QtCore.Signal(bool)
 
     toolitems = (
-        ('Home', 'Center display on contents', 'mdi.home', 'on_home_clicked', None),
+        ('Home', 'Reset axes limits', 'mdi.home', 'on_home_clicked', None),
```

Comment:
I think
```suggestion
        ('Home', 'Reset axis limits', 'mdi.home', 'on_home_clicked', None),
```
sounds better

Reply:
I think I prefer `axes` as it does affect both so I think the plural makes sense

Reply:
That's fair, I'll allow it 😉